docs/core/deploying/trimming-options.md

```diff
@@ -197,8 +205,43 @@ Several feature areas of the framework libraries come with linker directives tha
 
     Remove globalization specific code and data. For more information, see [Invariant mode](../run-time-config/globalization.md#invariant-mode).
 
+- `<MetadataUpdaterSupport>false</MetadataUpdaterSupport>`
+
+    Remove metadata update specific logic related to hot reload.
+
+- `<UseNativeHttpHandler>true</UseNativeHttpHandler>`
+
+    Use the default platform implementation of HttpMessageHandler for Android/iOS and remove the managed implementation.
+
 - `<UseSystemResourceKeys>true</UseSystemResourceKeys>`
 
     Strip exception messages for `System.*` assemblies. When an exception is thrown from a `System.*` assembly, the message will be a simplified resource ID instead of the full message.
 
  These properties will cause the related code to be trimmed and will also disable features via the [runtimeconfig](../run-time-config/index.md) file. For more information about these properties, including the corresponding runtimeconfig options, see [feature switches](https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md). Some SDKs may have default values for these properties.
+
+## Framework features disabled when trimming
+
+The following features are incompatible with trimming because they require code that is not statically referenced. These are disabled by default in trimmed apps.
+
+> [!WARNING]
+> Enable these features at your own risk. They are likely to break trimmed apps without extra work to preserve the dynamically referenced code.
+
+- `<BuiltInComInteropSupport>`
+
+   Built-in COM support is disabled.
+
+- `<CustomResourceTypesSupport>`
+
+   Use of custom resource types is not supported. ResourceManager code paths that use reflection for custom resource types is trimmed.
+
+- `<EnableCppCLIHostActivation>`
+
+    C++/CLI host activation is disabled.
+
+- `<EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization>`
```

Comment:
Hmm, it is interesting that `EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization` is disabled by default when you trim, but `EnableUnsafeBinaryFormatterSerialization` isn't.

Should `EnableUnsafeBinaryFormatterSerialization` also be set to `false` by default when you trim?

cc @GrabYourPitchforks @pgovind

Reply:
I've re-opened https://github.com/dotnet/sdk/issues/14475 for this issue.